hyperstart.go

```diff
@@ -315,6 +347,38 @@ func buildHyperContainerProcess(cmd Cmd) (hyperJson.Process, error) {
 	return process, nil
 }
 
+func isStarted(c net.Conn, chType chType) bool {
+	ret := false
+	timeoutDuration := 5 * time.Second
```

Comment:
5s is too much imho. If we can't ping in less than a second, we're not connected.